packaging/nnstreamer.spec

```diff
@@ -285,22 +285,25 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir
 
 ninja -C build %{?_smp_mflags}
 
-%if 0%{?unit_test}
     export NNSTREAMER_BUILD_ROOT_PATH=$(pwd)
```

Comment:
because `%if` statement is removed, I think it's better to fix the indents.

Reply:
Updated.